bin/fastRun.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -eu
+
+# source: https://stackoverflow.com/questions/59895/how-to-get-the-source-directory-of-a-bash-script-from-within-the-script-itself#246128
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+
+echo "Running directly, without checking/installing dependencies"
+node "${DIR}/node_modules/ep_etherpad-lite/node/server.js" "${@}"
```

Comment:
This won't work, because in this PR `fastRun.sh` is inside `bin`.

But it's really my fault, because in https://github.com/ether/etherpad-lite/issues/3711#issuecomment-599135831 I've indicated that I used `fastRun.sh` from `<base>/bin`, while really I use it from `<base>`.

I'll push the necessary corrections :)

Reply:
Done